### PR TITLE
Task/netcore

### PR DESCRIPTION
**Pull request by dnitsch**
#### 📲 What

tidy up netcore implementation

#### 🤔 Why
		
Why it's needed, background context.
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR
		 
#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: packages/scaffolding-cli/src/__tests__/domain/config/worker_maps/ssr_aks_tfs.test.ts [hunk 1]
```diff
@@ -3,31 +3,36 @@ import { BusinessSection, CloudSection } from "../../../../domain/model/prompt_a
...
-}
+} as CloudSection
 
-let files: Array<BuildReplaceInput> = [
+const files: Array<BuildReplaceInput> = [
     {
         files: ["**/package.json"],
         values: {
-            "PROJECT_NAME": projName
+            "PROJECT_NAME": projName,
```
Comment (estherthetesteramido) [inline packages/scaffolding-cli/src/__tests__/domain/config/worker_maps/ssr_aks_tfs.test.ts:27]: Duplication? Make it case insensitive?

File: packages/scaffolding-cli/templates/build/azDevops/azure/azure-pipelines-netcore-k8s.yml [hunk 2]
```diff
@@ -231,62 +229,61 @@ stages:
...
-                    kubernetes_clustername: $(aks_clustername)
+                    kubernetes_clusterrg: amido-stacks-nonprod-demo
+                    kubernetes_clustername: amido-stacks-nonprod-demo
                     # Mutation Information
-                    kubectl_filepath: $(Agent.BuildDirectory)/s/$(self_repo)/deploy/k8s/app/api-deploy.yml
-                    kubectl_working_dir: $(Agent.BuildDirectory)/s/$(self_repo)/deploy/k8s/app # unnecessary
-                    appinsights_enabled: false
-                    appinsights_accountName: ""
-                    secureFile: ""
+                    kubectl_filepath:
```
Comment (estherthetesteramido) [inline packages/scaffolding-cli/templates/build/azDevops/azure/azure-pipelines-netcore-k8s.yml:286]: @dnitsch just be careful of backwards compatibility for other pipelines using these tasks. Change the param type to a list breaks the other pipelines.

Comment (dnitsch) [reply packages/scaffolding-cli/templates/build/azDevops/azure/azure-pipelines-netcore-k8s.yml:286]: so generally the public facing generate files should all have this 
`ref: refs/tags/v1.x.x`  and the breaking change was lumped into a `v.1.3.0` tag so as long as all our current tests are using a version pegged to ~>v1.2 then it shouldn't be a problem

File: packages/scaffolding-cli/src/__tests__/domain/config/worker_maps/ssr_aks_tfs.test.ts (hunk 1 above)
Comment (dnitsch) [reply packages/scaffolding-cli/src/__tests__/domain/config/worker_maps/ssr_aks_tfs.test.ts:27]: yeah that would just increase the replace cycle time, rather would replace all occurences in caps into lower case for the node app case

**Pull request merged**